apps/middleman/src/lib/dal.ts

```diff
@@ -8,6 +8,8 @@ import {
   UserRole,
 } from "@/db/schema";
 import { eq } from "drizzle-orm";
+import { TestWorkflow } from "@igniter/middleman-workflows/workflows";
+import { getTemporalClient, getTemporalConfig } from "./temporal";
```

Comment:
```suggestion
```

See comments below for details. But in short, we temporal integration shouldn't be mixed with data-access operations. This is only a suggestion :grin: